core/trino-main/src/test/java/io/trino/sql/planner/assertions/PlanMatchPattern.java

```diff
@@ -1173,16 +1174,31 @@ public static GroupingSetDescriptor singleGroupingSet(List<String> groupingKeys)
         return new GroupingSetDescriptor(groupingKeys, 1, globalGroupingSets);
     }
 
+    public static SymbolMapper symbolMapper(SymbolAliases symbolAliases)
```

Comment:
make it private static method of `DynamicFilterPattern`

Reply:
done